src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -934,6 +915,45 @@ async def _genomic_to_tx_segment(
             genomic_ac, genomic_pos, is_seg_start, gene, tx_ac=transcript
         )
 
+    async def _select_optimal_transcript(
+        self, genomic_pos: int, genomic_ac: str, gene: str
+    ) -> str:
+        """Select the optimal transcript given a genomic position, accession, and gene.
```

Comment:
You might want to expand the description to state what "optimal" means in this case